src/skills-builder/skills-builder-modal/select-preferences/CareerInterestSelect.jsx

```diff
@@ -16,25 +18,24 @@ const CareerInterestSelect = () => {
   const { searchClient } = algolia;
 
   const handleCareerInterestSelect = (value) => {
-  // By checking for a value to exist, we avoid adding a null value to the careerInterests array
-  // The 'onSelected' function is fired during every 'onChange' event
-  // A null value was being passed to this function whenever the search box received input, resulting in empty cards
-    if (value && careerInterests.length < 3) {
+    if (!careerInterests.includes(value) || careerInterests.length < 3) {
```

Comment:
With the updates to `Form.Autosuggest` in Paragon, we no longer need to check for a `value`. However, we weren't checking for identical values before so that has also been added.